connector_mgmt.yaml

```diff
@@ -0,0 +1,814 @@
+openapi: 3.0.0 # need this as first line to allow some IDEs to know this is an openapi document.
+# All endpoints defined here will be discoverable by all users. If there is a need to keep some endpoints "private", add them to the "kas-fleet-manager-private.yaml" file
+info:
+  title: Connector Service Fleet Manager
+  version: 0.0.3
+  description: Connector Service Fleet Manager is a Rest API to manage connectors.
+servers:
+  - url: https://api.openshift.com
+    description: Main (production) server
+  - url: https://api.stage.openshift.com
+    description: Staging server
+  - url: http://localhost:8000
+    description: localhost
+  - url: /
+    description: current domain
+tags:
+  - name: Connector Types
+    description: ""
+  - name: Connectors
+    description: ""
+  - name: Connector Clusters
+    description: ""
+
+paths:
+  #
+  # These are the connector related APIs
+  #
+  '/api/connector_mgmt/v1/kafka_connector_types/{connector_type_id}':
```

Comment:
it should be `id` instead of `connector_type_id`

Reply:
Yeah. We can merge this as it is still alpha, and work to get those changes in, which should send us the updated spec.